code/game/objects/items/trash.dm

```diff
@@ -57,6 +57,10 @@
 	name = "energybar wrapper"
 	icon_state = "energybar"
 
+/obj/item/trash/bluespacebar
+	name = "bluespacebar wrapper"
```

Comment:
Inconsistent with how you wrote the name on 168

Reply:
hi @Crushtoe im the one who made the sprites and pathed the files, hence im the one who named the wrapper. I am not sure what you mean by inconsistent? the name is entirely consistent with other food items, for example: notice how the energy bars name is "High-powered energy bar" but its wrapper is called "energybar wrapper". I have opted to remain consistent with the energy bar naming convention and kept the bluespace bar wrapper as "bluespacebar wrapper".

if your issue is the inherent inconsistency with tg/stations wrapper naming convention then perhaps you can PR a change to make the names consistent across all food items?

Reply:
I will gladly consume the good boy points for grammar fix, thank you for your service

Reply:
I'll avoid doing that in future